http_16.go

```diff
@@ -0,0 +1,10 @@
+// +build go1.6
```

Comment:
How is this gonna work with Go 1.7? Do we have to update the build tags every new go release?